server/chat-plugins/blackjack.ts

```diff
@@ -485,7 +483,7 @@ export class Blackjack extends Rooms.RoomGame {
 		this.turnLog += turnLine;
 		if (player.cards.length > 2) this.display(turnLine, false, player.name);
 
-		if (player === this.dealer) {
+		if (player instanceof BlackjackDealer) {
```

Comment:
is instanceof hotpatchable?

Reply:
Given that the entire plugin would be hotpatched I think it'd be ok.